gaslift/GASLIFT-02.DATA

```diff
@@ -0,0 +1,1154 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+  
+-- Copyright (C) 2021 Equinor
+                                                                           

```

Comment:
-- This model is based on GASLIFT-01.DATA, it differs by introducing an irregular corner-point geometry.

Reply:
Done